docs/localization.md

```diff
@@ -98,7 +98,7 @@ function loadMessageBundle(config?: Config): Message {
 }
 
 export const nls = new Localization(
-  loadMessageBundle(JSON.parse(process.env.VSCODE_NLS_CONFIG))
+  loadMessageBundle(process.env.VSCODE_NLS_CONGIF ? JSON.parse(process.env.VSCODE_NLS_CONFIG!) : undefined)
```

Comment:
@vazexqi think that ! at the end of JSON.parse is an issue.

Reply:
@lcampos - Sorry, why would that be an issue? The bang is the non-null assertion operator. See https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator

There is an issue with `process.env.VSCODE_NLS_CONGIF` though, which I will fix.